packages/terra-site/src/App.jsx

```diff
@@ -60,36 +72,33 @@ class App extends React.Component {
 
   render() {
     const toggleContent = (
-      <Button className={styles['site-toggle']} icon={<IconMenu />} onClick={this.handleToggleClick} />
+      <CollapsibleMenuView.Item icon={<IconMenu />} isIconOnly key="toggle-content" onClick={this.handleToggleClick} />
     );
 
     const bidiContent = (
-      <ButtonGroup
-        className={styles['site-bidi']}
-        dir="ltr"
-        size="medium"
-        isSelectable
-        buttons={[
-          <ButtonGroup.Button isSelected text="ltr" key="ltr" onClick={() => document.getElementsByTagName('html')[0].setAttribute('dir', 'ltr')} />,
-          <ButtonGroup.Button text="rtl" key="rtl" onClick={() => document.getElementsByTagName('html')[0].setAttribute('dir', 'rtl')} />,
-        ]}
-      />
+      <CollapsibleMenuView.ItemGroup key="site-bidi" isSelectable dir="ltr" size="medium">
+        <CollapsibleMenuView.Item id="ltr" text="ltr" key="ltr" isSelected={this.state.isLtr} onClick={this.handleBidiChange} />
+        <CollapsibleMenuView.Item id="rtl" text="rtl" key="rtl" isSelected={!this.state.isLtr} onClick={this.handleBidiChange} />
+      </CollapsibleMenuView.ItemGroup>
     );
 
     const localeContent = (
-      <div className={styles['site-locale']}>
-        <label htmlFor="locale"> Locale: </label>
-        <select value={this.state.locale} onChange={this.handleLocaleChange}>
-          <option value="en">en</option>
-          <option value="en-GB">en-GB</option>
-          <option value="en-US">en-US</option>
-          <option value="de">de</option>
-          <option value="es">es</option>
-          <option value="fr">fr</option>
-          <option value="pt">pt</option>
-          <option value="fi-FI">fi-FI</option>
-        </select>
-      </div>
+      <CollapsibleMenuView.Item
+        text={`Locale: ${this.state.locale}`}
+        key="locale"
+        menuWidth="160"
+        shouldCloseOnClick={false}
+        subMenuItems={[
+          <CollapsibleMenuView.Item id="en" text="en" key="en" onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="en-GB" text="en-GB" key="en-GB" onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="en-US" text="en-US" key="en-US" onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="de" text="de" key="de" onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="es" text="es" key="es" onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="fr" text="fr" key="fr" onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="pt" text="pt" key="pt"onClick={this.handleLocaleChange} />,
+          <CollapsibleMenuView.Item id="fi-FI" text="fi-FI" key="fi-FI" onClick={this.handleLocaleChange} />,
```

Comment:
We should store the selected state to checkmark the selected option. 

![image](https://user-images.githubusercontent.com/6720991/31401002-160bab76-adb7-11e7-9d01-7eb4fd872a45.png)

Reply:
The [current implementation](https://github.com/cerner/terra-core/blob/master/packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx#L139) of CollapsibleMenu Subgroup Items does not carry forward a selection state. I think the limitation here is we are wanting select dropdown functionality via  submenu component.

Reply:
@emilyrohrbough Selection state would be set and maintained on the items passed through. If it's not working then it needs to be logged as a bug.
Also, I would have expected these options to be put in a selectable item group.

Reply:
`subMenuItems={[`
&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.ItemGroup isSelectable key="locale">`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="en" text="en" key="en" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'en'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="en-GB" text="en-GB" key="en-GB" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'en-GB'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="en-US" text="en-US" key="en-US" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'en-US'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="de" text="de" key="de" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'de'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="es" text="es" key="es" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'es'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="fr" text="fr" key="fr" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'fr'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`<CollapsibleMenuView.Item id="pt" text="pt" key="pt" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'pt'} />`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `<CollapsibleMenuView.Item id="fi-FI" text="fi-FI" key="fi-FI" onClick={this.handleLocaleChange} isSelected={this.state.locale === 'fi-FI'} />`
&nbsp;&nbsp;&nbsp;`</CollapsibleMenuView.ItemGroup>,`
`]}`

Reply:
The CollapsibleMenuView.Items are not selectable unless they are in an ItemGroup, otherwise you would need to use the CollapsibleMenuView.Toggle to get the checkmark styling.

Reply:
Ah. I did not consider placing these subitems into a single ItemGroup subitem